packages/db-objects-file/lib/objects/objectsInMemServerRedis.js

```diff
@@ -588,22 +558,28 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
 
         handler.on('exists', async (data, responseId) => {
             if (!data || !data.length) {
-                handler.sendInteger(responseId, 0);
+                return void handler.sendInteger(responseId, 0);
             }
 
             // Note: we only simulate single key existence check
             const {id, namespace, name} = this._normalizeId(data[0]);
 
             if (namespace === this.namespaceObj) {
+                let exists;
                 try {
-                    const exists = await this.objectExists(id);
-                    handler.sendInteger(responseId, +exists);
+                    exists = this._objectExists(id);
                 } catch (e) {
-                    return handler.sendError(responseId, e);
+                    return void handler.sendError(responseId, e);
                 }
+                handler.sendInteger(responseId, +exists);
             } else if (namespace === this.namespaceFile) {
-                // @ts-ignore
-                handler.sendInteger(responseId, +await this.fileExists(id, name));
+                let exists;
+                try {
+                    exists = this._fileExists(id, name);
+                } catch (e) {
+                    return void handler.sendError(responseId, e);
+                }
+                handler.sendInteger(responseId, +exists);
```

Comment:
Ugh, please no conversion hacks. Rather be explicit: `exists ? 1 : 0`

Reply:
;-) was there before :-) Will change it

Reply:
@foxriver76  :-))